firmware/hw_layer/adc/adc_subscription.cpp

```diff
@@ -95,6 +95,10 @@ static AdcSubscriptionEntry* findEntry() {
 	efiSetPadUnused(getAdcChannelBrainPin("adc unsubscribe", entry->Channel));
 #endif // EFI_PROD_CODE
 
+	if (entry->Sensor) {
```

Comment:
we don't need to use `entry->Sensor`, we can just use the `sensor` parameter (which should is `== entry->Sensor`)

Reply:
And then also don't need a branch.